data/console/test_java.sh

```diff
@@ -154,9 +156,17 @@ check_java_version_active_vs_dot () {
 	echo "check linked java version: PASS"
         #echo $dot_version_short
     else
-	echo "check linked java version: FAIL"
-        echo "linked java is: $java_version_active should be $dot_version_short"
-	exit 2
+        if [[ "$dot_version_short" == "1.7.1" ]]; then
+            # Special Case
+            if [[ "$java_version_active" == "1.7.0" ]]; then
+                echo "check linked java version: OK"
+                echo "INFO: linked java is: $java_version_active which is normal according to bnc#1014602"
+            fi
+        else
+	    echo "check linked java version: FAIL"
+            echo "linked java is: $java_version_active should be $dot_version_short"
```

Comment:
this part of the code is will be removed, thanks to the previous refactor. Yet again, thanks for catching this.